src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/ArrayTrailingCommaCheck.java

```diff
@@ -167,7 +167,7 @@ public void visitToken(DetailAST arrayInit) {
                 && !TokenUtil.areOnSameLine(rcurly, previousSibling)
                 && !TokenUtil.areOnSameLine(arrayInit, previousSibling)
                 && previousSibling.getType() != TokenTypes.COMMA) {
-            log(rcurly.getLineNo(), MSG_KEY);
+            log(rcurly, MSG_KEY);
```

Comment:
@romani 
@rnveach 
May be violation should be reported on last array item instead of rcurly?

Reply:
I am undecided.

Reply:
it is very hard to make decision, both is ok, lets keep on rcurly

Reply:
but if look at examples at https://checkstyle.org/config_coding.html#ArrayTrailingComma_Description we put "ok" and "violatation" on last element !!! it is good point that we need to make violation on last element.

@wilcoln , please change main code to report violation on lest element

Reply:
Roger that.

Reply:
Should we create a new Issue for this?